### DISCUSSION TITLE
A new xs breakpoint

### DISCUSSION THREAD

**Discussion by alihassan1133** [tailwindlabs/tailwindcss#2099] [Ideas]
It's difficult to style pages for really small devices (Galaxy S5 and iPhone 5/SE). Was trying to truncate a string using a `max-width` utility but text just overflowed off the page on those 2 devices.

**Comment by philippbosch** [comment]
Maybe check out the [`.truncate`](https://tailwindcss.com/docs/word-break/#truncate) and/or [`.overflow-*`](https://tailwindcss.com/docs/overflow/) utilities.

**Comment by alihassan1133** [comment]
@philippbosch Yes, I used `.truncate`. It worked fine on normal devices like iPhone 6/7/8+ but the page breaks on really small devices. I have 3 or 4 items in a flex row.

**Reply by philippbosch** [reply]
Here's how you can add a new breakpoint: https://tailwindcss.com/docs/breakpoints/
